tests/utils/test_builders.py

```diff
@@ -48,6 +48,27 @@ def test_tokenizer_builder(tokenizer_name: str, tokenizer_kwargs: dict):
         assert isinstance(tokenizer, PreTrainedTokenizerBase)
 
 
+@pytest.mark.parametrize('tokenizer_name,tokenizer_kwargs', [
+    ('bert-base-uncased', {
+        'model_max_length': 10,
+        'eos_token_id': None,
+    }),
+    ('bert-base-uncased', {
+        'model_max_length': 10
+    }),
+])
+def test_tokenizer_no_EOS(tokenizer_name: str, tokenizer_kwargs: dict):
+    tokenizer_kwargs = {
+        'model_max_length': 10,
+    }
+    tokenizer_name = 'bert-base-uncased'
+
+    with pytest.raises(ValueError) as exc_info:
+        build_tokenizer(tokenizer_name, tokenizer_kwargs)
+    assert 'must have an eos_token_id.' in str(
```

Comment:
I think instead of asserting a specific message separately, you can pass an expected message to `pytest.raises`